app/views/home/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="container">
   <div class="row text-center">
     <div class="col">
-      <h1>Vacinação Joinville Covid-19</h1>
+      <h1>Vacinação <%= Settings[:city_name] %> Covid-19</h1>
```

Comment:
We can safely switch this with "Vacinação Covid-19 CIDADE" to standardize it with the title 😄 

```suggestion
      <h1>Vacinação Covid-19 <%= Settings[:city_name] %></h1>
```